.circleci/config.yml

```diff
@@ -14,14 +14,132 @@ jobs:
+            PR_NUMBER=$(echo "$CIRCLE_PULL_REQUEST" | sed 's|.*/||')
+            RES=$( poetry run github-cli get-pr-base-sha \
+              --github-token "${GITHUB_TOKEN}" \
+              --repo "${REPO}" \
+              --pr-number "${PR_NUMBER}" )
+            #check if RES does not contain error, continue otherwise exit
+            if [[ $RES == *"error"* ]]; then
+              echo "Error: $RES"
+              exit 1
+            fi
+            export BASE_REV=$RES
+      - run:
+          name: Generate configs on base branch
+          working_directory: validation
+          command: |
+            echo "Base branch revision: $BASE_REV"
+            git checkout $BASE_REV
+            go run <<parameters.command>>/main.go base
+      - run:
+          name: Compute diff
+          working_directory: validation
+          command: |
+            ./scripts/diff-configs.sh <<parameters.directory>>/output-base <<parameters.directory>>/output-HEAD
+          when: always
+      - run:
+          name: If command failed, create or update comment on PR
+          when: on_fail
+          command: |
+            PR_NUMBER=$(echo "$CIRCLE_PULL_REQUEST" | sed 's|.*/||')
+            RUN_ID=${CIRCLE_WORKFLOW_ID}
+            REPO=${CIRCLE_PROJECT_REPONAME}
+            OWNER=${CIRCLE_PROJECT_USERNAME}
+            RUN_URL="https://app.circleci.com/pipelines/workflows/$RUN_ID"
+            BODY="<<parameters.message>>. You can see the detail [here]($RUN_URL)."
+
+            cd $GITHUB_CLI_PATH
+            COMMENT_ID=$( poetry run github-cli get-pr-comments \
+              --github-token "${GITHUB_TOKEN}" \
+              --repo "${REPO}" \
+              --pr-number "${PR_NUMBER}" \
+              --message-substring "${COMMENT_BODY}" \
+              ----user-type "Bot" )
+
+            if [ -n "$COMMENT_ID" ]; then
+              poetry run github-cli post-pr-comment \
+                --github-token "${GITHUB_TOKEN}" \
+                --repo "${REPO}" \
+                --pr-number "${PR_NUMBER}" \
+                --comment-body "${COMMENT_BODY}" \
+                --comment-id $COMMENT_ID
+            else
+              poetry run github-cli post-pr-comment \
+                --github-token "${GITHUB_TOKEN}" \
+                --repo "${REPO}" \
+                --pr-number "${PR_NUMBER}" \
+                --comment-body "${COMMENT_BODY}"
+            fi
+      - run:
+          name: If command passed, remove a comment on the PR if it exists
+          when: on_success
+          command: |
+            PR_NUMBER=$(echo "$CIRCLE_PULL_REQUEST" | sed 's|.*/||')
+            REPO=${CIRCLE_PROJECT_REPONAME}
+            OWNER=${CIRCLE_PROJECT_USERNAME}
+            # Delete the comment
 
 workflows:
   setup:
     jobs:
-       - setup-genesis-allocs-validation
+      - setup-genesis-allocs-validation
+
+  rollup-config-and-genesis-diff:
+    jobs:
+      - compute-config-diff-and-report-with-pr-comment:
```

Comment:
Do you want to give this job a different name to the other one?

Reply:
Comment on obsolete code